Bottom: ec82cc91919a4532353678a2db1afd9a73932bcb
Top:    0ec0a3522e435dece2e4abd709be380e722ad2a0
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-31 16:29:22 -0500

Refresh of buildbot-image-entrypoint

---
diff --git a/docker/buildbot-master/Dockerfile b/docker/buildbot-master/Dockerfile
index 76bd13be..e8005b9e 100644
--- a/docker/buildbot-master/Dockerfile
+++ b/docker/buildbot-master/Dockerfile
@@ -30,4 +30,5 @@ RUN adduser -h "/home/buildbot" -s "/bin/sh" -D -u ${BUILDBOT_UID} buildbot \
 &&  chown -v -R buildbot:buildbot "/home/buildbot"
 
 USER buildbot
+ENTRYPOINT ["/usr/bin/dumb-init"]
 CMD ["/home/buildbot/start.sh"]
